README.md

```diff
@@ -26,7 +26,13 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose(person='Jimmy')
+  puts "Would you take this rose and help out an old beggar, #{person}?"
+end
+
+def offerRose(person='young prince')
+  puts "Would you take this rose and help out a #{person}?"
+end
 ```
```

Comment:
You defined the method twice here. Only need to define once and then call it, so something like...

```rb
def offerRose( person )
  puts "Would you take this rose and help out an old beggar, #{person}?"
end

offerRose( "young prince" )
```